.gitignore

```diff
@@ -24,6 +24,7 @@ temp
 
 # Developers/ui_extensions
 content/en/developers/ui_extensions.md
+content/en/developers/faq/ui_extensions.md
```

Comment:
Leaving the old path to account for anyone who hasn't run a `clean-all` recently